### ISSUE TITLE
err=no volume plugin matched

### ISSUE THREAD

**Issue by qmfrederik** [kubernetes-csi/drivers#40]
I'm trying to create a CSI plugin for [nfusr](https://github.com/facebookincubator/nfusr), see https://github.com/quamotion/nfusr-csi.

When I provision a pod using a PVC which leverages the CSI plugin (see https://github.com/quamotion/nfusr-csi/blob/master/sample/nginx.yaml), the pod remains stuck in the `ContainerCreating` state, and the output of `systemctl status kubelet` on the node which hosts the pod is:

```
err=failed to get Plugin from volumeSpec for volume "data-nfusrplugin" err=no volume plugin matched
```

I'd love to troubleshoot this but I"m not sure where to start. The `csi-attacher-nfusrplugin` and `csi-nodeplugin-nfusrplugin` pods have been deployed and I don't see anything off in their output.

I'd be happy to provide any additional information.

**Comment by rootfs** [comment]
have u turned on CSI feature gate?

**Comment by qmfrederik** [comment]
I'm pretty sure I had, at least following the instructions specified here: http://blog.kubernetes.io/2018/01/introducing-container-storage-interface.html. 

I'm a bit time-constrained now so I reverted back to using user-space mounts in a sidecar container + mount propagation; but I'll probably give CSI plugins another try (either for nfusr or etcd) in the near feature.

Closing for now and I'll re-open/create a new issue if I hit any issues.
